vultr/terraform/vpc.tf

```diff
@@ -0,0 +1,6 @@
+resource "vultr_vpc" "this" {
```

Comment:
Please pick a better name for the resource instead of `this`.

Reply:
re-named